packages/expo-modules-autolinking/src/ExpoModuleConfig.ts

```diff
@@ -38,16 +38,16 @@ export class ExpoModuleConfig {
   }
 
   /**
-   * Returns a podspec path defined by the module author.
+   * Returns podspec paths defined by the module author.
    */
-  iosPodspecPath(): string | undefined {
+  iosPodspecPath(): string | string[] | undefined {
```

Comment:
What if we arrayize the results here? So it always returns an array of paths or `null/undefined`. Then we don't have to do this in other places where this function is used.
```suggestion
  iosPodspecPaths(): string[] | undefined {
```